src/layouts/DefaultLayout.vue

```diff
@@ -6,13 +6,22 @@
         :glossy="$q.theme === 'mat'"
         :inverted="$q.theme === 'ios'"
       >
-        <q-breadcrumbs active-color="white" color="light" v-if="paths.length > 0">
+        <q-breadcrumbs active-color="white" color="light">
+          <q-breadcrumbs-el  icon="home" label="Approve" to="/" />
           <q-breadcrumbs-el v-for="path in paths" :label="path" :key="path"/>
         </q-breadcrumbs>
-        <span v-else>
```

Comment:
what's this<!--{"state":"Active","line":12,"detailPos":{"start":{"line":12,"position":13},"end":{"line":14,"position":13}},"boxPos":{"top":50,"right":-30},"sha":"2616b1062b60d0dc8ec77e19eb8c0c598dce810a"}-->